src/App.tsx

```diff
@@ -1,9 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
+import { Movies } from './types/movies';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleChangeFilterText = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(e.target.value);
+  };
+
+  const visibleMovies = (moviesArray: Movies[], filter: string): Movies[] => {
+    const moviesCopy = [...moviesArray];
+
+    if (filter.length !== 0) {
```

Comment:
you can shorten it to 
```suggestion
    if (filter.length) {
```